### PR TITLE
adding additional alert rules to gcp privesc pack

### PR DESCRIPTION
**Pull request by mikiodehartj1**
### Alerts Rule Packs

- [ ] IF THIS CONTENT NEEDS TO BE RELEASED - is the package version in the package.json bumped?
- [ ] Does a related alert already exist, and should it be tweaked or added to instead?
- [ ] Test each query to make sure it works
- [ ] Look for hardcoded variables/parameter values in the query
- [ ] Consider Severity for Alerts
- [ ] Spellcheck
- [ ] Use all caps for J1QL keywords and relationship classes
- [ ] Upload the alerts rule pack JSON into JupiterOne to validate

### CODE REVIEW

File: rule-packs/gcp-privilege-escalation.json [hunk 1]
```diff
@@ -190,5 +190,77 @@
...
+        "name": "query0",
+        "query": "FIND google_iam_binding WITH permissions = 'coudbuilds.builds.create' AS permission THAT ASSIGNED >> (User | UserGroup) AS user RETURN user.displayName, user.email, permission.displayName ",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "gcp-possible-composer-get-privilege-escalation-path",
+    "description": "A potential attacker can use the following permissions to edit objects in storage buckets and update the environment, comprimising the composer's service account and associated permissions. It is recommended to limit which users or service accounts can create Composer jobs.",
```
Comment (SeaBlooms) [inline rule-packs/gcp-privilege-escalation.json:208]: compromising

**Review by SeaBlooms — CHANGES_REQUESTED** [review]
couple typos and then should be ready for merge

File: rule-packs/gcp-privilege-escalation.json [hunk 2]
```diff
@@ -190,5 +190,77 @@
...
     ],
     "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "gcp-possible-cloudbuilds-create-privilege-escalation-path",
+    "description": "Using the cloudbuilds.builds.create permission an attacker could potentially have access to additional read and write permissions to seven different GCP services (excluding Cloud Build itself). Most notably, they gain nearly-full access to Google Cloud Storage. It is recommended to restrict the permissions granted to the Cloud Build Service Account and to be careful granting the cloudbuild.builds.create permission to any users in your Organization.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND google_iam_binding WITH permissions = 'coudbuilds.builds.create' AS permission THAT ASSIGNED >> (User | UserGroup) AS user RETURN user.displayName, user.email, permission.displayName ",
```
Comment (erincrawford) [inline rule-packs/gcp-privilege-escalation.json:200]: Within the query (line 200), update 'coudbuilds.builds.create' to 'cloudbuilds.builds.create'

Comment (mikiodehartj1) [reply rule-packs/gcp-privilege-escalation.json]: fixed

**Review by SeaBlooms — APPROVED** [review]

**Pull request merged**